src/pip/_internal/network/session.py

```diff
@@ -225,16 +225,20 @@ class PipSession(requests.Session):
 
     timeout = None  # type: Optional[int]
 
-    def __init__(self, *args, **kwargs):
+    def __init__(
+        self,
+        *args,  # type: Any
+        retries=0,  # type: int
+        cache=None,  # type: Optional[str]
+        trusted_hosts=None,  # type: Optional[List[str]]
```

Comment:
Since `trusted_hosts` is never modified here, this can be made simpler:

```python
def __init__(
    ...
    trusted_hosts=(),  # type: Sequence[str]
    ...
):
    ...
    for host in trusted_hosts:
        self.add_trusted_host(host, suppress_logging=True)
```

Reply:
Thanks for the suggestion! Added in the latest revision.